components/bms_worker/src/BatteryMonitoring.c

```diff
@@ -417,3 +420,25 @@ void BMS_DischargeLimit(uint16_t relativeSoC) {
         BMS.dischargeLimit = 2.25f*relativeSoC; //linear function for the last 20% of discharge
     }
 }
+
+void BMS_HeatCurrentDischargeLimit(int16_t cellTemp) {
+    if (cellTemp >= 48) {
+        BMS.heatChargeCurrentLimit    = -3.75f * cellTemp + 225;
+    } else if (cellTemp > 60) {
+        BMS.heatDischargeCurrentLimit = 0;
+    } else {
+        BMS.heatDischargeCurrentLimit = MAX_CONTINOUS_DISCHARGE_CURRENT;
+    }
+}
+
+void BMS_HeatCurrentChargeLimit(int16_t cellTemp) {
+    if (cellTemp >= 48) {
```

Comment:
logic is faulty, if temp is above 60 you will never reach line 438

Reply:
i was thinking that it should be 0, if it gets above operating temps. That was just an assumption